lib/ex_unit/lib/ex_unit/cli_formatter.ex

```diff
@@ -210,14 +210,22 @@ defmodule ExUnit.CLIFormatter do
     do: colorize([:bright, :black], msg, config)
 
   defp formatter(:diff_delete, msg, config),
-    do: colorize(:red, msg, config)
+    do: colorize(:red, show_whitespace(msg), config)
 
   defp formatter(:diff_insert, msg, config),
-    do: colorize(:green, msg, config)
+    do: colorize(:green, show_whitespace(msg), config)
 
   defp formatter(_,  msg, _config),
     do: msg
 
+  defp show_whitespace(msg) do
+    if String.match?(msg, ~r/\A[ ]+\z/) do
```

Comment:
We can have a recursive function that also compacts multiple spaces into one fragment.
There is no need for `intersperse/2` step since `""` signals that it was a whitespace before thus we can put `" "` back into accumulator.